airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/formatter/BigDecimalUtil.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.formatter
+
+import java.math.BigDecimal
+import java.math.BigInteger
+
+object BigDecimalUtil {
```

Comment:
this class maybe belongs in the CDK?

Reply:
Seems reasonable. If we do that can we make this a singleton so we can mock it in tests?

Reply:
derp. didn't catch this comment before merging. will do this when I PR to master.